Find a functionally equivalent code:android.widget.TimePicker.setOnTimeChangedListener:COMMENT
Method Modifier: public      
Comment:/**
 * Set the callback that indicates the time has been adjusted by the user.
 *
 * @param onTimeChangedListener the callback, should not be null.
 */

@@ -1,3 +1,3 @@
 {
-    mOnTimeChangedListener = onTimeChangedListener;
+    mDelegate.setOnTimeChangedListener(onTimeChangedListener);
 }

